base-theme/assets/css/main.scss

```diff
@@ -54,27 +54,6 @@ h1 {
   }
 }
 
-h2 {
-  font-size: $heading-two-font-lg;
-  line-height: $heading-two-line-height-lg;
-  color: $black;
-
-  @include media-breakpoint-down(xs) {
-    font-size: $heading-two-font-xs;
-    line-height: $heading-two-line-height-xs;
-  }
-
-  &.branded {
-    font-family: Helvetica, sans-serif;
-    font-weight: 300;
-
-    span {
-      color: $orange;
-      font-weight: 600;
-    }
-  }
-}
-
```

Comment:
We're not using` class="h2"` anywhere in the code base now. The reason why I was compelled to remove this particular chunk of code was the use of `color: $orange`, since that was the old theme. We're not supposed to be using` $orange` as color anymore.

The other scss properties for other heading tags may also be due a removal/refactor since we're supposed to use semantic heading tags -- and I think we did refactored those classes a long time ago to use semantic headings. But the other scss properties (for removal or refactoring) is relatively more code which would need more testing. Leaving this as it is for now